### ISSUE TITLE
Unable to find setup plugin file

### ISSUE THREAD

**Issue by chrisblossom** [wallabyjs/quokka#40]
### Issue description or question

Unable to find custom plugin file located in project's root folder.

### Quokka.js file

```javascript
// package.json:

  "quokka": {
    "babel": true,
    "plugins": [
      "./quokka.js"
    ]
  }
```

### Sample repository link

*If the issue can not be reproduced just using the quokka file above (for example because it requires/imports some files from your project), please create a small repository where the issue can be reproduced.* 

### Quokka.js Console Output

```
Quokka #1 (node: v6.10.2, babel: v6.24.1)

Cannot find module './quokka.js'
	at Function.Module._resolveFilename module.js:469
	at Function.Module._load module.js:417
```

### Code editor version

WebStorm 2017.1.2
Build #WS-171.4249.40, built on April 25, 2017
JRE: 1.8.0_112-release-736-b21 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Mac OS X 10.12.4

### OS name and version

OSX 10.12.4

./quokka.js:
```javascript
'use strict';

const quokka = {
    before: () => {
        global.__DEVELOPMENT__ = true;
    },
};

module.exports = quokka;
```

**Comment by ArtemGovorov** [comment]
Just tried and can't reproduce it.

<img width="695" alt="screen shot 2017-04-28 at 11 45 52 pm" src="https://cloud.githubusercontent.com/assets/979966/25531388/dbeeb828-2c6c-11e7-8a0e-897d80b7b895.png">

Are there any special steps required to reproduce it?

**Comment by chrisblossom** [comment]
Looks like Quokka is looking for the plugin file relative to the file being executed instead of the project's root folder.

https://github.com/chrisblossom/quokka-issue-40

**Comment by ArtemGovorov** [comment]
The issue is fixed and the fix is published in Quokka plugin v1.0.2.
